src/clients/FtpGateway.cc

```diff
@@ -143,7 +143,8 @@ class Gateway : public Ftp::Client
     void checkUrlpath();
     void buildTitleUrl();
     void writeReplyBody(const char *, size_t len);
-    void printfReplyBody(const char *fmt, ...);
+    // unused
+    void printfReplyBody(const char *fmt, ...) PRINTF_FORMAT_ARG2;
```

Comment:
Please remove this change and post a separate official PR removing printfReplyBody() as unused (since 0477a07 AFAICT).